pdr_backend/subgraph/subgraph_payout.py

```diff
@@ -163,6 +163,7 @@ def fetch_payouts(
                 ),
                 "roundSumStakes": float(payout["prediction"]["slot"]["roundSumStakes"]),
                 "stake": float(payout["prediction"]["stake"]),
+                "pdr_slot_slot_id": f"{payout['id'].split('-')[1]}-{payout['id'].split('-')[2]}",
```

Comment:
The sot id format is `contract-slot` so the actual value is `"pdr_slot_slot_id": f"{payout['id'].split('-')[0]}-{payout['id'].split('-')[1]}"`